content/articles/2024-05-01-tiles-based-logs.md

```diff
@@ -0,0 +1,37 @@
+---
+url: /articles/tiles-based-logs
+title: "Tiles-Based Transparency Logs"
```

Comment:
Nitty one: How do you feel about "Tile-based"? "Tile**s**-based" scans slightly awkwardly for me and I can't think of any other noun-based adjectives which are plurals :)

Reply:
Easy fix, done!